04 - Guides, Workflows, & Courses/for Academic Writing.md

```diff
@@ -9,6 +9,8 @@ tags:
 # for Academics
 - [Comprehensive Academic Workflow from Reading to Writing in Markdown](https://www.notion.so/chrisgrieser/Comprehensive-Academic-Workflow-from-Reading-to-Writing-in-Markdown-a62298be91934043b11006be1ddc553a) by [[chrisgrieser|pseudometa]]
 - [Zotero -> zotfile -> mdnotes -> obsidian -> dataview Workflow](https://forum.obsidian.md/t/zotero-zotfile-mdnotes-obsidian-dataview-workflow/15536) by @Cat
+- [Mathematical Notes With Obsidian. Transforming Obsidian Into the Perfect Tool for STEM Notes](https://medium.com/beyond-productivity/using-mathjax-in-obsidian-c57640af11ec) by Beyond Boundaries
```

Comment:
I'm curious about the name "Beyond Boundaries" - the URL and page seem to suggest it might be "Beyond Productivity"???

Reply:
as far as I can tell, "Beyond Productivity" is the name of the blog, and "Beyond Boundaries" is the nickname of the author?
![image](https://user-images.githubusercontent.com/73286100/150639544-d9391201-19ed-40a8-8fa2-28fb67a7e8f2.png)

Reply:
Thanks - I definitely agree with that...